### DISCUSSION TITLE
v0.11.13

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#551] [Releases]
## What's Changed
* fixing bourne shell syntax error by @JtMotoX in https://github.com/alexjustesen/speedtest-tracker/pull/518
* Reduce the size of the published Docker image by @ottnorml in https://github.com/alexjustesen/speedtest-tracker/pull/532
* [Bugfix] Delete any on results policy by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/548
* Release v0.11.13 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/550

## New Contributors
* @JtMotoX made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/518
* @ottnorml made their first contribution in https://github.com/alexjustesen/speedtest-tracker/pull/532

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.11.12...v0.11.13

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.11.13'>v0.11.13</a>.</em>

**Comment by zackmuc** [comment]
Just checked it for the bug #548 I created another user, for the default user it works for the created not. Is there a fix possible?

**Reply by alexjustesen** [reply]
Any user should have access to delete results. It's based on this policy so I'd try logging out and back in again to see if that helps.

https://github.com/alexjustesen/speedtest-tracker/blob/main/app/Policies/ResultPolicy.php
